google/cloud/storage/client.cc

```diff
@@ -59,7 +59,7 @@ ObjectMetadata Client::UploadFileSimple(
   std::string payload(std::istreambuf_iterator<char>{is}, {});
   request.set_contents(std::move(payload));
 
-  return raw_client_->InsertObjectMedia(request).second;
+  return ThrowOnError(raw_client_->InsertObjectMedia(request));
```

Comment:
What's your philosophy on including a header that is already included/used in this file's header but which is used independently here?

Reply:
I am lazy and prefer not to add additional `#includes` until needed.  I know that forces us to fix some files when a include is removed and was used indirectly.

Having said that I do not feel too strongly about it one way or the other.

If you prefer "include what you use" I am Okay with it, but "somebody" will need to setup the tools to enforce it.

Reply:
I don't feel strongly about this case either.  Mostly I'm just trying to get a feel for what you'd like to see comments on.  (I'm sure there will be more of that for a while.)

I probably would feel more strongly about a header including what it uses, but we should be able to "enforce" that by having any test for that interface include the header before anything else.